front_end_pocket_poll/lib/groups_widgets/group_page.dart

```diff
@@ -41,6 +43,45 @@ class _GroupPageState extends State<GroupPage>
   final int votingTab = 2;
   final int occurringTab = 1;
   final int closedTab = 4;
+  final Map<int, int> listIndexesToEventTypes = new Map<int, int>();
```

Comment:
Would be helpful to explain what this mapping is used for as a comment.

Reply:
I added a comment to explain what this map is used for.